src/report.sh

```diff
@@ -289,9 +300,10 @@ function show_report()
 function save_data_to()
 {
   local path="$1"
+  local flag="$2"
```

Comment:
You should add a line
```bash
  flag=${flag:-'SILENT'}
```
to prevent verbose mode of being the default for this function (see `cmd_manager` definition if you don't understand why).

That is the reason why you had to alter a case in the `test_save_data_case` unit test, which shouldn't happen as (in this case of the unit test) we are calling `save_data_to` without passing `flag`.